kwasm-lemmas.md

```diff
@@ -325,8 +347,19 @@ They are non-trivial in their implementation, but the following should obviously
       requires WIDTH ==Int 1
       [simplification]
 
-    rule #getRange(BM, ADDR, WIDTH) modInt 256 => #get(BM, ADDR)
-      requires notBool (WIDTH ==Int 0)
+    rule #getRange(BM, ADDR, WIDTH) modInt BYTE_SIZE => #get(BM, ADDR)
+      requires BYTE_SIZE ==Int 256
+       andBool notBool (WIDTH ==Int 0)
+       andBool #isByteMap(BM)
+      [simplification]
+
+//  rule #getRange(BM, ADDR, WIDTH) => #getRange(BM, ADDR +Int 1, WIDTH -Int 1)
+//    requires #get(BM, ADDR) ==Int 0
+//     andBool WIDTH >Int 0
+
```

Comment:
let's not merge commented out lemmas. You can keep in your own tree

Reply:
Fixed